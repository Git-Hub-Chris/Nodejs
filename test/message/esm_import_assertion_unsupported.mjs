import '../common/index.mjs';
import 'specifier' assert { type: 'unsupported' };
