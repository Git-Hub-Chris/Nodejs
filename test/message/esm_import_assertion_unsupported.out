node:internal/errors:*
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_IMPORT_ASSERTION]: Invalid "type" import assertion: "unsupported"
    at new NodeError (node:internal/errors:*:*)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:*:*)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:*:*)
    at link (node:internal/modules/esm/module_job:*:*) {
  code: 'ERR_INVALID_IMPORT_ASSERTION'
}

Node.js *
